judgels-backends/uriel/uriel-app/src/main/resources/migrations/001.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="aguss787" id="Add-points-for-problem">
+        <addColumn catalogName="judgels_uriel" schemaName="judgels_uriel" tableName="uriel_contest_problem">
+            <column name="points" type="INT" defaultValue="0">
```

Comment:
If it is nullable, then the default value should be null? (or not specified at all)

Reply:
Sorry, my bad :|